api/src/main/java/org/openmrs/module/openconceptlab/client/OclClient.java

```diff
@@ -125,8 +132,68 @@ public GetMethod executeExportRequest(String url, String latestVersion) throws I
 		return exportUrlGet;
 	}
 
+	/**
+	 * This constructs the url to import concepts from the subscription url provided
+	 * @param url the subscription url
+	 * @param latestVersion latest released version of the specified collection
+	 * @return GetMethod configured with the export url
+	 */
+	public GetMethod getExportUrl(String url, String latestVersion){
+		GetMethod exportUrlGet;
+
+		url = removeLastUrlForwardSlashIfExist(url);
+
+		if(url.contains(latestVersion)){
+			exportUrlGet = new GetMethod(url + "/export");
+		}else {
+			exportUrlGet = new GetMethod(url + "/" + latestVersion + "/export");
+		}
+		return exportUrlGet;
+	}
+
+	/**
+	 * This gets the desired collection release version
+	 * @param url the subscription url
+	 * @param token the subscription API token
+	 * @return the retrieved collection version
+	 * @throws IOException
+	 */
+	public String  getOclReleaseVersion(String url, String token) throws IOException {

```

Comment:
Where is the IOException thrown in this method?

Reply:
https://github.com/openmrs/openmrs-module-openconceptlab/blob/98eb2764b6fd564fd6be40daae19a8f12f0544c8/api/src/main/java/org/openmrs/module/openconceptlab/client/OclClient.java#L178